en/django_forms/README.md

```diff
@@ -415,7 +415,7 @@ $ git push
 
 {% filename %}command-line{% endfilename %}
 ```
-$ cd my-first-blog
+$ cd ~/yourusername.pythonanywhere.com
```

Comment:
If you really want the include the domain there, you can use ```$USER``` instead of ```yourusername``` I think.

Reply:
I hadn't thought of that!  yes.  although maybe a bit too much like magic?

Reply:
the `$USER.pythonanywhere.com` change would definitely work, and it would work nicely for the places we use `workon` to activate the virtualenv as well.

it still feels a bit wrong to me somehow.  both the cd and the workon will tab-complete, which should mitigate any copy-paste problems... But I'm happy to go with what y'all prefer on this one.  Let me know and I'll make the change.